src/lineup.test.js

```diff
@@ -0,0 +1,25 @@
+const test = require('ava');
+const sinon = require('sinon');
+const axios = require('axios');
+
+const lineup = require('./lineup');
+const tvheadendApi = require('./tvheadendApi');
+
+test('calls the API with the right options', async t => {
+  const expectedResponse = {
+    entries: [{
+      enabled: true,
+      number: 1,
+      name: 'Test Channel',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
      name: "Test Channel",
```